src/components/PluginDependencies.jsx

```diff
@@ -27,9 +27,18 @@ function PluginDependencies({dependencies} ) {
             </div>
         );
     };
+    const reverseDependencyLink = (dependency) => {
+        return (
+            <div key={`${dependency.node.id}-revese`}>
```

Comment:
revese?

```suggestion
             <div key={`${dependency.node.id}-reverse`}>
```

or key only needs to be unique in the array its in so you could even just use id

```suggestion
             <div key={dependency.node.id`}>
```